.github/workflows/build-image.yml

```diff
@@ -63,7 +71,7 @@ jobs:
           platforms: linux/amd64,linux/arm64
           provenance: false
           builder: ${{ steps.buildx.outputs.name }}
-          push: ${{ github.repository == 'mastodon/mastodon' && github.event_name != 'pull_request' }}
+          push: ${{ github.event_name != 'pull_request' }}
```

Comment:
I'm curious if there's a hard reason to disable push on PRs, at least for ghcr.io?